src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -107,13 +112,16 @@ public Person toModelType() throws IllegalValueException {
         }
         final Address modelAddress = new Address(address);
 
-        if (remark == null) {
-            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Remark.class.getSimpleName()));
-        }
-        final Remark modelRemark = new Remark(remark);
+        final Remark modelRemark = new Remark(Optional.ofNullable(remark).orElse(""));
```

Comment:
@NewtonKoh FYI I made Remark optional in the json file because it was causing all the contacts to disappear.
![image](https://github.com/AY2324S2-CS2103T-T16-2/tp/assets/40530851/c7d9dc7e-e41c-470e-99e4-af24735d2f81)